.github/workflows/test-template.yml

```diff
@@ -141,8 +142,15 @@ jobs:
           # Fails if set to true
           skip-tsc: false
 
+      - name: Get Preview Features
+        id: gpf
+        run: ./.github/workflows/scripts/get-preview-features.js
+        env:
+          PREVIEW_FEATURE_SET: ${{ matrix.previewFeatureSet }}
+          EXCLUDED_PREVIEW_FEATURES: 'driverAdapters'
```

Comment:
Driver adapters are a special feature, so in our "normal" tests, we don't enable it yet to keep things separated, also because they are tested in a separate job.